tests/click_tests/test_autolock.py

```diff
@@ -110,10 +111,10 @@ def test_autolock_passphrase_keyboard(device_handler: "BackgroundDeviceHandler")
     assert layout.text == "< PassphraseKeyboard >"
 
     CENTER_BUTTON = buttons.grid35(1, 2)
-    # total sleep time: 11 * 1.1 = 12.1 seconds
-    for _ in range(11):
+    # keep clicking for long enough to trigger the autolock if it incorrectly ignored key presses
+    for _ in range(math.ceil(11 / 1.5)):
         debug.click(CENTER_BUTTON)
-        time.sleep(1.1)
+        time.sleep(1.5)
 
     debug.click(buttons.OK, wait=True)
     assert device_handler.result() == "mzAZ4BgqmFHYxhPgdFH2pR2h1X7jJrNFSs"
```

Comment:
i'm assuming this won't be true anymore now that the number of presses comes out different?

Reply:
uhhhhhnnnnrrg [6685c10](https://github.com/trezor/trezor-firmware/pull/2806/commits/6685c10cbbe6dc2653279b08bf0558e096b52778)